multiversx_sdk/adapters/query_runner_adapter.py

```diff
@@ -60,3 +59,32 @@ def run_query(self, query: SmartContractQuery) -> SmartContractQueryResponse:
             return_message=adapted_query_response.return_message,
             return_data_parts=adapted_query_response.get_return_data_parts()
         )
+
+
+class ContractQuery:
```

Comment:
Kept for this PR, understood. Though, must be dropped before v1, as soon as this is ready:

https://github.com/multiversx/mx-sdk-specs/pull/70